CHANGELOG.md

```diff
@@ -72,6 +72,7 @@ Ref: https://keepachangelog.com/en/1.0.0/
 
 ### Improvements
 
+* [#14691](https://github.com/cosmos/cosmos-sdk/pull/14691) Change behavior of `sdk.StringifyEvents` to do not flatten events attributes by events type.
```

Comment:
Are log used in the state machine?

Reply:
I think it is worth mentioning that this only affects the `Log` string of the tx execution result, not the `Events`

Reply:
Okay, is it okay to include in a point version, then? Or better directly in v0.47?
Someone _could_ query the logs and store it right?

Reply:
If you ask me, this does not belong in a patch release. I think clients also deserve stable interfaces.

Reply:
> If you ask me, this does not belong in a patch release. I think clients also deserve stable interfaces.

Didn't think about that sorry. Yeah, that makes sense then.